api/src/main/java/org/openmrs/CohortMembership.java

```diff
@@ -94,15 +94,15 @@ public void setPatientId(Integer patientId) {
 	}
 	
 	public Date getStartDate() {
-		return startDate;
+		return new Date(startDate.getTime());
```

Comment:
@ribhavsharma please have a look at [this](http://www.informit.com/articles/article.aspx?p=31551&seqNum=2)

You will have to use `clone()` method while returning the value which will also fix travis-ci build failure :)